src/Symfony/Bridge/Doctrine/ArgumentResolver/EntityValueResolver.php

```diff
@@ -61,20 +49,20 @@ public function supports(Request $request, ArgumentMetadata $argument): bool
         }
 
         $options = $this->getOptions($argument);
-        if (null === $options['class']) {
+        if (!$options->class) {
             return false;
         }
 
-        if ($options['attribute_only'] && !$options['has_attribute']) {
+        if (!$this->defaults->autoConvert && !$options->hasAttribute) {
```

Comment:
autoConvert might be clearer, also to ease the migration from FEB

Reply:
What's about `if (!$this->defaults->autoConvert) {`

- When people use the default option `autoConvert = false` they can add an annotation `#[MapEntity()]` to enable the behavior on desired routes
- When people use the default option `autoConvert = true` they can add an annotation `#[MapEntity(autoConvert: false)]` to disable the behavior on desired routes

This solves the issue reported by @derrabus: When people want to disable the auto-conversion on a few routes, they have to disable the feature on the entire application and add annotation on all the routes.

Reply:
Good idea, updated, and renamed to `map`.